irc_commands.c

```diff
@@ -499,6 +499,30 @@ static void irc_cmd_invite( irc_t *irc, char **cmd )
 		irc_send_num( irc, 341, "%s %s", iu->nick, ic->name );
 }
 
+static void irc_cmd_kick( irc_t *irc, char **cmd )
+{
+	irc_channel_t *ic;
+	irc_user_t *iu;
+	
+	if( ( iu = irc_user_by_name( irc, cmd[2] ) ) == NULL )
+	{
+		/* KICK does not support ERR_NOSUCHNICK */
```

Comment:
Tested with 6 ircds over 8 networks (even ircnet and efnet), all of them behave like that. PRETTY SURE IT'S SAFE